.github/mergify.yml

```diff
@@ -0,0 +1,80 @@
+---
+defaults:
+  actions:
+    queue:
+      name: default
+      method: rebase
+      update_method: rebase
+
+
+# each test should be listed separately, do not use regular expressions:
+# https://docs.mergify.io/conditions.html#validating-all-status-check
+# TODO: Use mergify's recently added 'shared configuration support'
+# to dedup some of the check-x=y repetition in the future.
+queue_rules:
+  - name: default
+    conditions:
+      - check-success=check
+      - check-success=build
+      - check-success=podmanbuild
+      - check-success=dockerbuild
+      - check-success=test
+      - check-success=test-kubernetes
+      - check-success=centos-ci/sink-clustered/mini-k8s-1.25
+
+        - check-success=dpulls
+
+
+pull_request_rules:
+  # Clearing approvals after content changes
+  - name: Remove outdated approvals
+    conditions:
+      - base=master
+    actions:
+      dismiss_reviews:
+        approved: true
+        changes_requested: false
+  # Perform automatic merge on conditions
+  - name: Automatic merge on approval
+    conditions:
+      - check-success=check
+      - check-success=build
+      - check-success=podmanbuild
+      - check-success=dockerbuild
+      - check-success=test
+      - check-success=test-kubernetes
+      - check-success=centos-ci/sink-clustered/mini-k8s-1.25
+
+      - "-draft"
+      # Contributors should set the 'do-not-merge' label if they don't want
+      # the PR to be (auto)merged for some reason.
+      - "label!=do-not-merge"
+      # A reviewer should set a label starting with 'review-in-progress' (and
+      # suffixed by their username) in order to indicate a detailed review has
+      # been started and not completed. This will hold the PR until the
+      # label has been removed.
+      - "-label~=^review-in-progress"
+      - "base=master"
+      # Even if there are 2 or more approvals we won't automerge if there are
+      # any changes requested.
+      - "#changes-requested-reviews-by=0"
+      - or:
+      # Any contributor's PR can be automerged with 2 (or more) reviews.
```

Comment:
Following warning should go away if we indent the comment properly.

Reply:
> Following warning should go away if we indent the comment properly.
It is  gone now, @anoopcs9 . the checks have all passed in the CI.

Reply:
It's a warning not an error, so the CI shows green but it's still showing up if you view the patch on github.

Reply:
Correct. It's a warning and thus test passes. While we are at it let's just fix it now.

Reply:
I agree with @anoopcs9. Please fix this when you have a moment

Reply:
> 



> I agree with @anoopcs9. Please fix this when you have a moment

Trying to. and I think the commentindentation errors or warnings are gone now, but I see the following locally from yamllint and in the editor:

```
56:6      error    syntax error: expected <block end>, but found '<block sequence start>' (syntax)
  57:7      error    wrong indentation: expected 7 but found 6  (indentation)
```

trying to fix all of this now ...

Reply:
I don't understand the block sequence error abd how to fix it. the rest should be taken care of now ...